### DISCUSSION TITLE
Allow usage of labels templating in when an aalert rule uses a classic condition.

### DISCUSSION THREAD

**Discussion by der-ali** [grafana/grafana#53481] [Alerting]
In order to get useful alerts notification i was searching for a way to add variables to the alert description in the notification.

I was so glad to found [this page](https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/variables-label-annotation/), but got really disappointed as i figured out that **using `$labels` variable is unavailable when the rule uses a classic condition** which we primarily use.

It would be very helpful to have this feature!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
